pkg/go-sqlsmith/stateflow/walker.go

```diff
@@ -212,6 +221,22 @@ func (s *StateFlow) walkExprNode(node ast.ExprNode, table *types.Table, column *
 	return nil
 }
 
+func (s *StateFlow) walkHintList(hintLength int, table *types.Table) (*types.Table, []*ast.TableOptimizerHint) {
+	hList := make([]*ast.TableOptimizerHint, 0)
+	hintNames := make(map[string]bool, 0)
+	for i := 0; i < hintLength; i++ {
+		if h := builtin.GenerateHintExpr(table); h != nil {
+			// remove duplicated hints
+			if _, ok := hintNames[h.HintName.String()]; !ok {
+				hList = append(hList, h)
+				hintNames[h.HintName.String()] = true
+			}
+		}
+	}
+	// TODO: remove conflict hints
```

Comment:
What will be if there are conflict hints?

Reply:
onw of two confict hints may be ignored...